cmd/gardener-extension-provider-local/app/app.go

```diff
@@ -328,7 +328,7 @@ func (w *webhookTriggerer) Start(ctx context.Context) error {
 		return err
 	}
 
-	return w.trigger(ctx, w.client, w.client, nil, &appsv1.DeploymentList{}, client.MatchingLabels{"app": "dependency-watchdog-probe"})
+	return w.trigger(ctx, w.client, w.client, &appsv1.DeploymentList{}, client.MatchingLabels{"app": "dependency-watchdog-prober"})
```

Comment:
```suggestion
	return w.trigger(ctx, w.client, w.client, nil, &appsv1.DeploymentList{}, client.MatchingLabels{"app": "dependency-watchdog-prober"})
```

Reply:
I had made this change but forgot to commit it . Pls find it in [e3b04f6](https://github.com/gardener/gardener/pull/6693/commits/e3b04f64bbc71415ffc2606260615cfa5ee15aaf)